NEWS.md

```diff
@@ -1,14 +1,22 @@
-# metabonaut 0.0.4 
+# metabonaut 0.0
```

Comment:
Is the missing .5 on purpose ? I think i changed it before but I'm wondering that maybe i'm doing something wrong aha